cli/local/errors.go

```diff
@@ -113,6 +114,14 @@ func ErrorPortAlreadyInUse(port int) error {
 func ErrorUnableToFindAvailablePorts() error {
 	return errors.WithStack(&errors.Error{
 		Kind:    ErrUnableToFindAvailablePorts,
-		Message: fmt.Sprintf("unable to find available ports"),
+		Message: "unable to find available ports",
+	})
+}
+
+func ErrorBindDockerInDocker(err error) error {
```

Comment:
Here's what the error looks like:

<img width="719" alt="Screen Shot 2020-09-21 at 4 38 32 PM" src="https://user-images.githubusercontent.com/808475/93831907-ea6b1880-fc28-11ea-9186-97a5b63da6da.png">